### PR TITLE
[style] Add economic benefits box #115

### PR DESCRIPTION
**Pull request by EthanGu4**
### Description
[//]: # "[Required] Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
- Added economic benefits variable with type string/null
- Created new component in ProjectModal/index.tsx with the same name
- Created new styling components named EconomicBenefits and EconomicBenefitsContainer
- Made small changes in font-color

### Screenshots
[//]: # "Add any screenshots that will help your PR reviewer understand what’s going on in the PR!"
[//]: # "[Required] For frontend tasks: add screenshots of the layout and expected behavior (GIFs if you’re feeling extra fancy)!"
[//]: # "Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

## How to review
[//]: # "/[Required] Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?"
- Review types/schema.d.ts
- Review ProjectModal/index.tsx and ProjectModal/styles.ts
- Finally, review minor changes in styles/texts.ts
- Review most recent commit

## Next steps
[//]: # "[Optional] - What has not yet been added to the PR, doesn’t yet have any functionality — essentially, what needs to be fixed or added."
[//]: # "[Optional] any temporary fixes in this PR that should be cleaned up later."

- check font sizes, doesn't match the one proposed in the figma due to wrong font-weights in styles/texts.ts

### Online sources
[//]: # "[Optional] insert links to any useful tutorials or documentation that you used for this PR!"

### Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @jjstnlee

### CODE REVIEW

**Review by jjstnlee — APPROVED** [review]
great job.

File: components/ProjectModal/styles.ts [hunk 1]
```diff
@@ -190,14 +190,36 @@ export const AdditionalInfo = styled.div`
...
+
+export const EconomicBenefits = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 16.95rem;
+  padding: 1.25rem 1.4rem;
+  border-radius: 8px 8px 12px 12px;
+  border: 1px solid rgba(46, 58, 89, 0.05);
+  box-shadow:
+    0px 2px 4px 0px rgba(77, 87, 114, 0.08),
```
Comment (nehaahussain) [inline components/ProjectModal/styles.ts:203]: nit: I noticed that the other components (KDM, technology, last updated, etc) don't have this box shadow on line 203 but economic benefits and additional information do. @jjstnlee does this matter to u... do we want to make things cohesive (small problem ik lol)
Screenshot 2025-03-04 at 7 03 10 PM

**Review by nehaahussain — APPROVED** [review]
other than my comment, everything looks good to me! great job gugu:)

File: components/ProjectModal/styles.ts (hunk 1 above)
Comment (jjstnlee) [reply components/ProjectModal/styles.ts:203]: we can talk to madison abt this, i think its inconsistent on the figma as well for some reason!
